spark/src/test/scala/geotrellis/spark/io/file/FileTileFeatureSpaceTimeSpec.scala

```diff
@@ -0,0 +1,28 @@
+package geotrellis.spark.io.file
+
+import geotrellis.raster.{Tile, TileFeature}
+import geotrellis.spark._
+import geotrellis.spark.io._
+import geotrellis.spark.io.index._
+import geotrellis.spark.testfiles.TestTileFeatureFiles
+
+import com.github.nscala_time.time.Imports._
+
+
+class FileTileFeatureSpaceTimeSpec
+    extends PersistenceSpec[SpaceTimeKey, TileFeature[Tile, Tile], TileLayerMetadata[SpaceTimeKey]]
+    with SpaceTimeKeyIndexMethods
+    with TestEnvironment
+    with TestTileFeatureFiles
+//    with CoordinateSpaceTimeTests
```

Comment:
This has been un-commented.  I went ahead and did the same in `FileSpaceTimeSpec`, as well.